src/options/quantifiers_options.toml

```diff
@@ -1992,9 +1992,17 @@ header = "options/quantifiers_options.h"
   category   = "regular"
   long       = "print-inst-full"
   type       = "bool"
-  default    = "true"
+  default    = "false"
   help       = "print instantiations for formulas that do not have given identifiers"
 
+[[option]]
+  name       = "debugInst"
+  category   = "regular"
```

Comment:
This should arguably be an `expert` option.

Reply:
This is actually a feature request from a user :)

Reply:
Ok, SG :)